examples/julia-machine-learning/MNIST-example.ipynb

```diff
@@ -0,0 +1,480 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "4f705426-0538-4858-9681-f60159298780",
+   "metadata": {},
+   "source": [
+    "# Train a Simple Machine Learning Model Using Julia"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55af3953-a4ac-421d-8fa0-5ff3cffb27ea",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-03-07T14:49:27.748000Z",
+     "iopub.status.busy": "2022-03-07T14:49:27.237000Z",
+     "iopub.status.idle": "2022-03-07T14:49:28.718000Z",
+     "shell.execute_reply": "2022-03-07T14:49:28.718000Z"
+    }
+   },
+   "source": [
+    "![Julia Logo](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/julia-logo.png \"doc-image\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a1daeeb-2625-406d-a2f8-ebd3e9038864",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This example describes how to run a simple machine learning workflow using the famous MNIST dataset of handwritten digits. We will be labeling images of numbers. \n",
```

Comment:
Confirm my words are correct! I would say that the intro here is a little sparse so I am trying to expand it a bit. Feel free to expand it more.

```suggestion
    "This example describes how to run a neural learning workflow using the famous MNIST dataset of handwritten digits. Flux.jl is a powerful Julia library for many types of machine learning, including neural networks. We will training a neural network on images of handwritten numbers to create an image classifier. \n",
```